keyboards/unicomp/mini_m/info.json

```diff
@@ -0,0 +1,113 @@
+{
+    "keyboard_name": "Mini M",
+    "manufacturer": "Unicomp",
+    "url": "",
+    "maintainer": "purdeaandrei",
+    "usb": {
+        "vid": "0x16C0",
+        "pid": "0x27DB",
+        "device_version": "0.0.1"
+    },
```

Comment:
```suggestion
    },
    "processor": "RP2040",
    "bootloader": "rp2040",
    "board": "GENERIC_RP_RP2040",
    "matrix_pins": {
        "ghost": true,
        "io_delay": 1
    },
    "indicators": {
        "on_state": 0,
        "num_lock": "GP22",
        "caps_lock": "GP21",
        "scroll_lock": "GP20"
    },
    "bootmagic": {
        "matrix": [0, 1]
    },
```

Reply:
DONE manually (because I have two levels on info.json. I expect other controllers to become available in the future for this keyboard, so I have placed the controller-specific settings in the inner info.json)